app/templates/editions/octane.hbs

```diff
@@ -14,12 +14,52 @@
     It has modern, streamlined components and state management that make it fun to build web applications.
     With seamless interoperability for existing apps, teams can migrate at their own pace, while developers building new apps start out with the best that Ember has to offer.
   </p>
-  <ul class="unstyled-list grid lg:grid-5">
-    <EsLinkCard @href="https://blog.emberjs.com/2019/12/20/octane-is-here.html" @title="Read the announcement" />
-    <EsLinkCard @href="https://guides.emberjs.com/release/getting-started/quick-start/" @title="Try the new app Quick Start" />
-    <EsLinkCard @href="https://guides.emberjs.com/release/tutorial/" @title="Follow the Tutorial" />
-    <EsLinkCard @href="https://guides.emberjs.com/release/" @title="Read the Guides" />
-    <EsLinkCard @href="https://guides.emberjs.com/release/upgrading/current-edition" @title="Learn how to upgrade" />
+  <ul class="lg:grid-5">
```

Comment:
**praise**

The changes look great (how you formatted code consistently, in particular)!